src/connectionController.ts

```diff
@@ -511,11 +511,11 @@ export default class ConnectionController {
   }
 
   public getActiveConnectionDriverUrl(): string | null {
-    if (!this._connectingConnectionId) {
+    if (!this._currentConnectionId) {
       return null;
     }
 
-    return this._savedConnections[this._connectingConnectionId].driverUrl;
```

Comment:
It doesn't do a connection itself, it just gets the connection name for code lense. I just tried these two ids `this._connectingConnectionId` and `this._currentConnectionId` and the playground behaves identically. How to reproduce your problem?

Reply:
It's not the playground, it's the shell launcher: Command + Shift + P, "Launch MongoDB Shell".

Reply:
Oh got it!